misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/core/TestAnnotationProcessor.java

```diff
@@ -314,23 +238,30 @@ public void process(FunctionNode functionNode, List<AnnotationAttachmentNode> an
     // Extract mock function information
     @Override
     public void process(SimpleVariableNode simpleVariableNode, List<AnnotationAttachmentNode> annotations) {
```

Comment:
It's better if we can divide this method since its complexity is high and long.

Reply:
Further divided the method for better understandability